docs/src/pages/components/data-grid/editing/CatchEditingEventsGrid.tsx

```diff
@@ -22,7 +21,7 @@ export default function CatchEditingEventsGrid() {
 
   React.useEffect(() => {
     return apiRef.current.subscribeEvent(
-      GRID_CELL_EDIT_START,
+      GridEvents.cellEditStart,
```

Comment:
We should be using the string value here. 

```suggestion
      'cellEditStart',
```

The same for `GridEvents.cellEditStop` below.